vignettes/mapping_summary.Rmd

```diff
@@ -17,25 +17,46 @@ library(knitr)
 
 # Summary
 
-This package contains many-to-many mappings between local authority districts (UTLA and LTLA) and NHS Acute Trusts in England, based on COVID-19 hospital admissions:
+This package contains two many-to-many mappings between local authority districts (UTLA and LTLA) and NHS Acute Trusts in England:
 
 * Trust-UTLA mapping: `covid19.nhs.data::trust_utla_mapping`
 * Trust-LTLA mapping: `covid19.nhs.data::trust_ltla_mapping`
 
-These mappings can be used to map data reported by local authority (e.g. testing) to NHS Trusts, or to map Trust-level data (e.g. admissions) to local authorities.
-
-The mappings contain the following variables:
+These mappings each contain the following variables:
 
 * `trust_code`: the three-digit organisation code for NHS Trusts.
 * `geo_code`: the nine-digit identifier for local authorities (UTLA or LTLA) in England
-* `p_trust`: the proportion of admissions to a given Trust were admitted from a given local authority (UTLA or LTLA)
-* `p_geo`: the proportion of admissions from a given local authority (UTLA or LTLA) that were admitted to a given Trust.
+* `p_trust`: the estimated proportion of admissions to a given Trust were admitted from a given local authority (UTLA or LTLA)
+* `p_geo`: the estimated proportion of admissions from a given local authority (UTLA or LTLA) that were admitted to a given Trust.
+
+These mappings are probabilistic estimates based on total COVID-19 discharges until 30 September. These mappings can be used to estimate COVID-19 hospital admissions at the local authority level, or to estimate "relevant" community COVID-19 cases for an NHS Trust. These are estimates only and may not accurately reflect the truth; see limitations below.
+
+
+# Methods
+
+The mappings in this package are derived from a raw mapping provided by NHS England, based on the Secondary Uses Service (SUS) healthcare data for England. This raw mapping counts the number of COVID-19 hospital discharges of between 01 January 2020 and 30 September 2020 from an NHS site (hospital) to lower-tier local authorities (LTLAs).
+
+The mappings in this package are summaries of the raw mapping. Site-LTLA pairs are aggregated to Trust-LTLA and Trust-UTLA pairs and to maintain anonymity we excluded any Trust-UTLA or Trust-LTLA pairs where the number of discharges is less than 10. Finally, we report only the proportion of admissions by Trust (`p_trust`) or local authority (`p_geo`).
+
+The mapping includes the following Trust mergers that have taken place since January 2020:
+
+```{r mergers}
+
+readr::read_csv(here::here("data-raw", "raw", "england_trusts", "trust_mergers.csv")) %>%
```

Comment:
Probably worth replacing with web links so they work when the package is installed but the repository is not local. Potentially just a flag about what this needs in order to run. This might cause CRAN CI checks to fail as is.

Reply:
Done in #2504288